packages/react/src/ReactHooks.js

```diff
@@ -33,19 +33,17 @@ export function useContext<T>(
       const realContext = (Context: any)._context;
       // Don't deduplicate because this legitimately causes bugs
```

Comment:
This comment is no longer relevant — deduplication refers to warnings. Instead there should be a justification for DEV-only invariant here.